### PR TITLE
Clarify exact date for requiring Python 3 support

### PR DESCRIPTION
**Pull request by astrofrog**
Just a small wording change to the affiliated package guidelines

### CODE REVIEW

File: documents/affiliated_package_review_guidelines.md [hunk 1]
```diff
@@ -164,7 +164,7 @@ compared to what one might realistically achieve.
 <table>
 <tr>
 <td width=150><img src="https://img.shields.io/badge/Red-red.svg" alt="Red"></td>
-<td>Nothing is 'unacceptable' at this point, but once after 2020 ‘Only compatible with Python 2’ goes from orange to red.</td>
+<td>Nothing is 'unacceptable' at this point, but once after 1 January 2020 ‘Only compatible with Python 2’ goes from orange to red.</td>
```
Comment (kelle) [inline documents/affiliated_package_review_guidelines.md]:
```suggestion
<td>Nothing is currently 'unacceptable'. Starting 1 January 2020, ‘Only compatible with Python 2’ will be red.</td>
```

Comment (kelle) [reply documents/affiliated_package_review_guidelines.md]: Also add on line 171 what Orange will change to: <td>Only compatible with Python 2. Starting 1 January 2020: 'Not fully Python 3'</td>. (I just made up the text...I'm not sure what Orange should be.)

Comment (taldcroft) [reply documents/affiliated_package_review_guidelines.md]: Maybe we should take the approach of astropy core and effectively remove all references to Py2 (as if it never existed).  So the words here would be simply "Compatible with Python 3" or "Not compatible with Python 3".

Comment (taldcroft) [reply documents/affiliated_package_review_guidelines.md]: Leading to:
```
<td>Nothing is currently 'unacceptable'. Starting 1 January 2020, 
    ‘Not compatible with Python 3’ will be red.</td>
```
On a different note, is the weird (non-ASCII?) single quote in the second bit intended?

Comment (astrofrog) [reply documents/affiliated_package_review_guidelines.md]: @taldcroft @kelle - thanks for the suggestions! I've gone with @taldcroft's latest suggestion. The non-ASCII quote has been removed.

**Pull request merged**